pkg/scheduler/admission/podnodeconstraints/admission.go

```diff
@@ -77,6 +78,9 @@ var resourcesToCheck = map[unversioned.GroupResource]unversioned.GroupKind{
 // we choose not to handle in this plugin
 var resourcesToIgnore = []unversioned.GroupKind{
 	extensions.Kind("DaemonSet"),
+	securityapi.Kind("PodSecurityPolicySelfSubjectReview"), // TODO: should this go through admission?
```

Comment:
> OK. I'm going to do. For some reasons I thought this could be done in the next PR.

If its small, I'd like it here, if its not, we can defer.  Just let me know.